Add changelog for 0.70.0

rgrunber opened the pull request:
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

@fbricon

testforstephen commented on CHANGELOG.md:
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.70.0 (November 3rd, 2020)
```
I just merged PR for issue https://github.com/eclipse/eclipse.jdt.ls/issues/1202, could you please update it to the changelog in both vscode-java and eclipse.jdt.ls sides.

Since RC bits is not ready, update release date to Nov. 4th to give a buffer for testing?

rgrunber replied:
I've added #1202 and set the date to Nov 4th.

testforstephen reviewed (commented):
Once you update the change log, feel free to merge it.

Eskibear commented:
Are you going to include #1676  and #1679 in this release or the next one?

testforstephen commented:
#1676  and #1679 are merged, need add it to the changelog.

fbricon commented on CHANGELOG.md:
```diff
@@ -1,5 +1,13 @@
+ * bug fix - GTD is not working if referenced library is updated without file name change. See [JLS#1577](https://github.com/eclipse/eclipse.jdt.ls/issues/1577).
+ * bug fix - Method references are given the CompletionItemKind.Module type by the completion provider. See [#1661](https://github.com/redhat-developer/vscode-java/issues/1661).
+ * other - Update TP to 2020-12 M1. See [JLS#1567](https://github.com/eclipse/eclipse.jdt.ls/issues/1567).
+ * other - Add the java.configuration.maven.globalSettings property. See [#1365](https://github.com/redhat-developer/vscode-java/issues/1365).
```
enhancement

rgrunber replied:
Done.

fbricon commented on CHANGELOG.md:
```diff
@@ -1,5 +1,13 @@
+ * enhancement - Convert a lambda expression to method reference. See [#1448](https://github.com/redhat-developer/vscode-java/issues/1448).
+ * bug fix - GTD is not working if referenced library is updated without file name change. See [JLS#1577](https://github.com/eclipse/eclipse.jdt.ls/issues/1577).
+ * bug fix - Method references are given the CompletionItemKind.Module type by the completion provider. See [#1661](https://github.com/redhat-developer/vscode-java/issues/1661).
+ * other - Update TP to 2020-12 M1. See [JLS#1567](https://github.com/eclipse/eclipse.jdt.ls/issues/1567).
```
Update Target Platform to Eclipse 2020-12 M1

fbricon commented:
Needs #1668 too

rgrunber commented:
> #1676 and #1679 are merged, need add it to the changelog.

I've added #1676 and used #1678 (which references the issue resolved by #1678)

> Needs #1668 too

Added as well.

I think Fred is expecting to merge one more thing and then we can do a final update, and prepare the build for testing tomorrow.

rgrunber commented on CHANGELOG.md:
```diff
@@ -1,5 +1,18 @@
+ * bug fix - Changing java.semanticHighlighting.enabled does not consistently update semantic tokens. See [JLS#1678](https://github.com/redhat-developer/vscode-java/issues/1678).
+ * bug fix - Method references are given the CompletionItemKind.Module type by the completion provider. See [#1661](https://github.com/redhat-developer/vscode-java/issues/1661).
+ * other - Update Target Platform to Eclipse 2020-12 M1. See [JLS#1567](https://github.com/eclipse/eclipse.jdt.ls/issues/1567).
+* other - Pass the cancellationtoken to the client request registered by provider. See [#1668](https://github.com/redhat-developer/vscode-java/pull/1668).
```
We can fix this later on.
